3_javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,146 @@
+console.log("-----------------------------When will we be there??-----------------------------")
+const travelInformation = {
+    speedT: 50, //km/h
+    speed: 50 / 60, //km/min  ====>  speed = 5/6 km/min
+    destinationDistance: 432, //min
+  };
+
+function timeCalculation(speed , displacement){
+    travelTimeInMinute = 432 * 6 / 5 ; //min
+    const houres = Math.floor (518.4 / 60) ;
+    const minutes = travelTimeInMinute - (houres * 60);
+    return `this journey will take ${houres} houres and ${minutes} minutes to arrive to the destination.`
+}
+
+  const travelTime = timeCalculation(5/6 , 432);
```

Comment:
and here you do not have to hardcode values as well, you can : 
`const travelTime = timeCalculation(travelInformation.speed, travelInformation.destinationDistance);
console.log(travelTime);`